tm2py/model/assignment/skim_transit_network.py

```diff
@@ -0,0 +1,1565 @@
+import inro.modeller as _m
+import inro.emme.database.emmebank as _eb
+import inro.emme.desktop.app as _app
+import inro.emme.core.exception as _except
+import inro.emme.desktop.worksheet as _worksheet
+import traceback as _traceback
+from copy import deepcopy as _copy
+from collections import defaultdict as _defaultdict, OrderedDict
+import contextlib as _context
+
+
+import multiprocessing as _multiprocessing
+import argparse as _argparse
+import re as _re
+import time as _time
+import numpy
+import os as _os
+import json as _json
+from copy import deepcopy as _copy
+
+
+try:
+    import openmatrix as _omx
+
+    def open_file(file_path, mode):
+        return OmxMatrix(_omx.open_file(file_path, mode))
+
+
+except Exception as e:
+    import omx as _omx
+
+    def open_file(file_path, mode):
+        return OmxMatrix(_omx.openFile(file_path, mode))
+
+
+# _all_periods = ['AM']
+_all_periods = ["ea", "am", "md", "pm", "ev"]
```

Comment:
should be params?

Reply:
and params should be ALL_CAPS and not private

Reply:
The skim_transit_network.py is taken directly from travel-model-two transit-ccr and needs some updates to match the standards of this repo. However, this project is ongoing so I prefer to wait to make sure there are no other incoming changes / bug fixes which are missed.